plugin/src/shared/sc/plugin2021/GameState.kt

```diff
@@ -22,35 +26,33 @@ class GameState @JvmOverloads constructor(
         /** Das zweite Team, @see [Team]. */
         override var second: Player = Player(Team.TWO),
         /** Der Spielstein, der in der ersten Runde gesetzt werden muss. */
-        @XStreamAsAttribute val startPiece: PieceShape = GameRuleLogic.getRandomPentomino(),
-        /** Das aktuelle Spielfeld. */
-        @XStreamAsAttribute override val board: Board = Board(),
+        @XStreamAsAttribute
+        val startPiece: PieceShape = GameRuleLogic.getRandomPentomino(),
         turn: Int = 0,
         /** Der zuletzt gespielte Zug. */
         override var lastMove: Move? = null,
         /** Speichert für jede Farbe, die alle Steine gelegt hat, ob das Monomino zuletzt gelegt wurde. */
-        @XStreamAsAttribute
-        val lastMoveMono: MutableMap<Color, Boolean> = mutableMapOf()
+        val lastMoveMono: HashMap<Color, Boolean> = HashMap(),
+        /** Das aktuelle Spielfeld. */
+        override val board: Board = Board(),
```

Comment:
Won't removing the XStream annotation from board result in the board not being serialized?

Reply:
No, XStream serializes all fields by default. That annotation serializes the field as XML field rather than subtag, which isn't possible for the Board - so it was a noop.

Reply:
Okay, thanks for clearing this up!